test/functionalTest/cases/0950_GET_v2_entity/GET_v2_entity.test

```diff
@@ -95,37 +95,37 @@ Date: REGEX(.*)
             "contextElement": {
                 "attributes": [
                     {
-                        "name": "speed",
```

Comment:
I understand that there aren't actual changes in this file, one trailing withespace. Is that correct?